### DISCUSSION TITLE
Add children list when item has children

### DISCUSSION THREAD

**Discussion by anthonyivol** [payloadcms/plugin-nested-docs#26] [Show and tell]
I've made a fork to add a children list after the breadcrumb field. 
It is sketchy at the moment but it may be an interesting feature : https://github.com/anthonyivol/plugin-nested-docs

<img width="860" alt="image" src="https://github.com/payloadcms/plugin-nested-docs/assets/19983700/17f642c8-48a4-4ecc-b317-9ae850a5ee91">

**Comment by Vethavear** [comment]
Second this! I would ditch Blog Category Collection and just rely on BlogPost if BlogPost could list its children
